src/core/service.js

```diff
@@ -42,8 +44,8 @@ const transform = (actionType, value, store) => {
 
 /* eslint-disable no-console */
 function doCallEndpoint(callSpec, next, onReceive, onError, store) {
-  const { method, path, urlParams, body, token } = callSpec
-  return apiFetch(compileUrl(path, urlParams), fetchOptions(method, body, token))
+  const { method, path, urlParams, body, token, baseUrl } = callSpec
+  return apiFetch(compileUrl(path, urlParams), { ...fetchOptions(method, body, token), baseUrl })
```

Comment:
You were wondering if we should allow to set the baseUrl per request, and I think that you already implemented this, since that "callSpec" can be created by the user, if he includes baseUrl in the call, then it will override those options of the "configOpts".

This relies on the order of the merge here 

```{ ...configOpts.call, ...callFromAction(store, action) },```

As the action call comes AFTER it will override the global value.

Makes sense to me